src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -711,6 +718,9 @@ StDebugger >> initializeShortcuts: aWindowPresenter [
 StDebugger >> initializeStack [
 
 	stackTable := self newTable.
+	stackTable 
+		activateOnDoubleClick;
+		whenActivatedDo: [ :selection | self doBrowseClass ].
```

Comment:
This shouldn't be related to this PR.

(I don't manage to find the related issue about that)